ddtrace/vendor/debtcollector/removals.py

```diff
@@ -57,14 +57,23 @@ class removed_property(object):
 
     # Message templates that will be turned into real messages as needed.
     _PROPERTY_GONE_TPLS = {
-        'set': "Setting the '%s' property is deprecated",
-        'get': "Reading the '%s' property is deprecated",
-        'delete': "Deleting the '%s' property is deprecated",
+        "set": "Setting the '%s' property is deprecated",
+        "get": "Reading the '%s' property is deprecated",
+        "delete": "Deleting the '%s' property is deprecated",
     }
 
-    def __init__(self, fget=None, fset=None, fdel=None, doc=None,
-                 stacklevel=3, category=DeprecationWarning,
-                 version=None, removal_version=None, message=None):
+    def __init__(
```

Comment:
we should avoid all of these formatting changes

Reply:
Fixed